core/templates/dev/head/pages/topics_and_skills_dashboard/SkillsListDirective.js

```diff
@@ -130,6 +133,10 @@ oppia.directive('skillsList', [
                     ).then(function() {
                       $rootScope.$broadcast(
                         EVENT_TOPICS_AND_SKILLS_DASHBOARD_REINITIALIZED);
+                    }).then(function(){
```

Comment:
```suggestion
                    }).then(function() {
```

Reply:
(Missed this one, sorry. Everything else is good!)

Reply:
Done